roles/openshift_management/defaults/main.yml

```diff
@@ -5,6 +5,10 @@ openshift_management_project: openshift-management
 openshift_management_project_description: CloudForms Management Engine
 # Number of retries when waiting for the app to start (retried every 30 seconds)
 openshift_management_pod_rollout_retries: 30
+# Node selector for project:
+cfme_default_projects:
```

Comment:
Not sure how this dict would work. Currently it would always assign a default node selector? Do we want different nodeselectors to different CFME projects?

Reply:
I based it off of this: https://github.com/openshift/openshift-ansible/blob/8c6192e10192b75af79b82e5e9f21cdc560a6e14/roles/openshift_hosted/tasks/create_projects.yml

And that I just added in the description would be just for allowing the app to be deployed on certain nodes using the NodeSelector.  However, you are allowed to have more than one CloudForms deployment, so if they wanted to segment it out to different CFME projects, it would work just as well in either case wouldn't it?

Reply:
Ok, got it. In this case `default` should be replaced with `openshift-management` so that it would work with default settings

Reply:
Lets stick to `openshift_` prefix here. Also, CFME creates just one project, I think it would be simplier to create just `openshift_management_nodeselector` var and set it to infra nodes by default

Reply:
Hello @vrutkovs I have made the changes you asked for and set the default for `region=infra`.

Reply:
Hey @vrutkovs , wouldn't this PR also need to accompany a [doc change](https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_management/README.md
) since we're changing the deployment of the where it was (it was free to go anywhere, now we're configuring to to region=infra per your request)

Reply:
Hey @vrutkovs , just checking in to see if you need anything form my end since it's been a while.